sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/ProtoDescriptorGenerator.scala

```diff
@@ -48,7 +48,7 @@ private[impl] object ProtoDescriptorGenerator {
 
     val protoBuilder = DescriptorProtos.FileDescriptorProto.newBuilder
     protoBuilder
-      .setName(name + ".proto") // FIXME: snake_case this ?!
+      .setName(packageName.replace(".", "/") + "/" + name + ".proto") // FIXME: snake_case this ?!
```

Comment:
I could do sth with the snake_case fixme, but I'm not if it's worth it.

Reply:
Maybe some context would be useful to review it. During the discovery process, we are deduplicating file descriptors in `kalix.javasdk.impl.AnySupport#flattenDescriptors`. Without the package prefix, the first component with a given name wins.

Reply:
If we don't tackle [the snake case](https://protobuf.dev/programming-guides/style/#file-structure) I wouldn't even do the `replace`.

Reply:
I think I added this FIXME. Not sure what I was thinking then. I was probably trying to be overly precise (and failing) on the generation and had the intention to make it look like a real proto file. 

Do we need to append `.proto`? If not, I would simply use the FQCN without bothering with replacing the dots. This is for internal usage, so we don't need to comply to any convention.

If we need the extension, then using `/` is preferable as it should look like a file location.